RealmSwift/Sync.swift

```diff
@@ -306,6 +306,16 @@ public struct SyncCredentials {
     public static func jwt(_ token: Token) -> SyncCredentials {
         return SyncCredentials(RLMSyncCredentials(jwt: token))
     }
+    
+    /// Initialize new credentials using a nickname.
+    public static func nickname(_ nickname: Token, _ isAdmin: Bool = false) -> SyncCredentials {
```

Comment:
Why did you opt to omit the argument label for the `isAdmin` argument? It makes it unclear what the boolean argument at the call site means. I think it should be included unless there's a strong reason not to.

Shouldn't the nickname argument be of type `String`? I realize that `Token` is a type alias for `String` so it's sorta pedantic, but aren't we expecting the user to pass an arbitrary string of their choosing here rather than some form of token?

Reply:
No you're right. It should be String, non-nil.